driver/random.hpp

```diff
@@ -91,6 +91,7 @@ inline T gen_0_to_B(T B)
 template <typename T>
 inline T gen_A_to_B(T A, T B)
 {
+    assert(B >= A);
```

Comment:
```suggestion
    assert(B > A);
```
I guess B must not be equal to A, otherwise we may get integer division by 0.

But interesting fact - while B < A does not make much sense, it will work fine for signed integers, since `X%(Y)==X%(-Y) `, but won't work for unsigned integers (at all) and floats (will behave strangely).

Reply:
good point. I was thinking about that myself. Initially I though B was inclusive to the range, but now it's clear that it's exclusive.